walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -24,19 +24,48 @@
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, 
                                      MeetingRequest request) {
-    // Get unavailable time for all required attendees in meeting request
     ArrayList<TimeRange> unavailable = new ArrayList<>();
+
+    // Get unavailable time for all required attendees in meeting request
     for (String attendee : request.getAttendees()) {
       ArrayList<TimeRange> meetingTimes = getMeetingTimes(events, attendee);
-      merge(unavailable, meetingTimes);
+      mergeTo(unavailable, meetingTimes);
     }
 
     // Get time ranges that are longer than the requested meeting duration
+    // for required attendees
     Collection<TimeRange> result = getAvailability(unavailable);
     result.removeIf( 
       timeRange -> timeRange.duration() < request.getDuration());
```

Comment:
per https://screenshot.googleplex.com/nMLdzbk8fES, use +4 spaces for indentation.